Dockerfile-dev

```diff
@@ -1,4 +1,4 @@
-FROM clearmatics/zeth-base:latest
+FROM clearmatics/zeth-base:v0.0.0
```

Comment:
Why did you switch to `v0.0.0`? We want to always build the `dev` and subsequent images on top of the `zeth-base:latest` image on master.

Reply:
I would like to recommend you avoid using tag `latest` by the reasons:
* local docker, kubernetes nodes, compose etc cache images if the tags the same. 
* versions allow us to track source of images

If this points not applicable for `dev` image building I can roll it back to tag `latest`

Should I do it?

Reply:
reverted

Reply:
Yes, it's handy to build from `latest` for the `dev` image (we build the dev image on some of our tests on the CI server where we fetch the latest image everytime the CI runs). Thanks for reverting.